Kickstarter-iOS/Views/Controllers/PaymentMethodsViewController.swift

```diff
@@ -176,3 +175,37 @@ extension PaymentMethodsViewController: AddNewCardViewControllerDelegate {
     self.viewModel.inputs.cardAddedSuccessfully(message)
   }
 }
+
+private extension UIView {
```

Comment:
I have copy pasted these from Justin's other PR and made them `private`. This way when all of our branches meet up we can simply remove these private extensions and use a global (public) extension.